### PR TITLE
Admin can upload casa_org logo #1088

### LINKED ISSUE #1088: Admin can upload casa_org logo
AS an **admin**,
I WANT to be able to upload my `casa_org`'s logo,
SO THAT my `casa_org` instance is branded and familiar to staff and volunteers. 

### Acceptance Criteria
On the Edit CASA Organization page,
there is a button to choose and upload an image file.
One uploaded, this image file populates in the header for this `casa_org`.

### PR DESCRIPTION
**Pull request by rebecarancan**
### What github issue is this PR for, if any?
Resolves #1088

### What changed, and why?
- Installed Active Storage to manage file uploads
- Installed gem image_processing to handle images  easily
- Added a file upload field on Casa Org form
- The logo uploaded will appear in the sidebar

### How will this affect user permissions?
- Volunteer permissions:
- Supervisor permissions:
- Admin permissions:

### How is this tested? (please write tests!) 💖💪
- Added a expectation on CasaOrg model spec to test the attach functionality
- Added a expectation on CasaOrg edit request spec to test if the upload was successful

### Screenshots please :)
**CasaOrg form:**
Captura de tela de 2020-10-19 18-26-06

**New CasaOrg logo on sidebar:**
Captura de tela de 2020-10-19 18-26-55

Captura de tela de 2020-10-19 18-01-58

### CODE REVIEW

File: app/views/layouts/_sidebar.html.erb [hunk 1]
```diff
@@ -1,13 +1,11 @@
...
+  <div class="sidebar-heading text-center">
+    <% if all_casa_admin_signed_in? || !current_organization.logo.attached? %>
       <%= image_pack_tag("media/src/images/default-logo.png",
           alt: "CASA Logo",
           class: "d-inline-block align-text-bottom") %>
     <% else %>
-      <%= image_pack_tag(current_organization.logo_url,
-          alt: current_organization.logo_alt_text || "CASA Logo",
-          class: "d-inline-block align-text-bottom") if current_organization&.logo_url %>
+      <%= image_tag current_organization.logo.variant(resize_to_limit: [200, 100]) %>
```
Comment (littleforest) [inline app/views/layouts/_sidebar.html.erb:8]: Please keep the `alt` text.

**Review by littleforest — CHANGES_REQUESTED** [review]

**Comment by compwron** [comment]
This is looking very very good! 
+1 please put back the alt text and then I think we're ready to merge! :) 💪✨

**Comment by littleforest** [comment]
@compwron please hold off on merging, I want to have @rebecarancan add one line to the Dockerfile so imagemagick will be installed but wanted to test locally first! 

@rebecarancan thanks for getting this set up!  Also, please fix merge conflicts when you get a chance.

**Comment by littleforest** [comment]
@rebecarancan could you add imagemagick to [this section](https://github.com/rubyforgood/casa/blob/master/Dockerfile#L31-L37) of the Dockerfile like so:

```
RUN apk update && apk upgrade && apk add --update --no-cache \
  bash \
  imagemagick \
  nodejs \
  postgresql-client \
  tzdata \
  vim \
  yarn && rm -rf /var/cache/apk/*
```

**Review by littleforest — APPROVED** [review]
🎉 💯

**Pull request merged**